pages/index.vue

```diff
@@ -15,27 +15,34 @@
           />
         </nuxt-link>
         <DButton
-            name="Go to Events Calendar"
-            variant="dark"
-            size="medium"
-            v-scroll-to="'#scroll'"
-          />
+          v-scroll-to="'#scroll'"
+          name="Go to Events Calendar"
+          variant="dark"
+          size="medium"
+        />
       </template>
     </DHero>
     <section class="d-hero is-small">
       <div class="hero-body">
         <div class="container">
-          <h1 role="heading" aria-level="1" class="d-calendar-title" id='scroll'>Events</h1>
+          <h1
+            id="scroll"
```

Comment:
name the `id` more idiomatically - maybe `events` then the `v-scroll-to` above will read more clearly as well